challenges/desafio28.js

```diff
@@ -0,0 +1,18 @@
+db.voos
+  .find({
+    $and: [
+      { "empresa.nome": "LATAM AIRLINES BRASIL" },
+      { natureza: "Doméstica" },
+    ],
+  })
+  .count();
+
+db.resumoVoos.insertOne({
+  empresa: "LATAM AIRLINES BRASIL",
+  totalVoosDomesticos: 20026,
+});
```

Comment:
uma outra forma de fazer aqui seria usando subquerie diretamente na chave totalVoosDomesticos, ficaria mais ou menos assim.
```suggestion

db.resumoVoos.insertOne({
  empresa: "LATAM AIRLINES BRASIL",
  totalVoosDomesticos: db.voos
  .count({
    $and: [
      { "empresa.nome": "LATAM AIRLINES BRASIL" },
      { natureza: "Doméstica" },
    ],
  }),
});
```